generator/config/expression/case.yaml

```diff
@@ -0,0 +1,21 @@
+# $schema: ../schema.json
+name: $case
+link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/switch/'
+type:
+    - resolvesToAny
```

Comment:
It should resolve to a new type "switchCase", as it can't be used anywhere an expression is expected.

Reply:
Added a new `SwitchBranch` interface for this and defined it as a type in the generator.